MetadataMigration/src/test/java/org/opensearch/migrations/commands/EvaluateTest.java

```diff
@@ -24,5 +38,21 @@ void evaluate_failsUnexpectedException() {
         var results = meta.evaluate(args).execute(context);
 
         assertThat(results.getExitCode(), equalTo(Evaluate.UNEXPECTED_FAILURE_CODE));
+        assertThat(results.getErrorMessage(), equalTo("Unexpected failure: host is null"));
+    }
+
+    @Test
+    void evaluate_failsUnexpectedExceptionInnerMessage() {
+        var args = new EvaluateArgs();
+        var meta = new MetadataMigration();
+        var context = mock(RootMetadataMigrationContext.class);
+ 
+        var evaluate = spy(meta.evaluate(args));
+        doThrow(new RuntimeException("Outer", new RuntimeException("Inner"))).when(evaluate).createClusters();
```

Comment:
As per the above, I'll make the argument that when you have a RuntimeException that isn't a known more specific type, you definitely should NOT be eating the stacktraces.